advocacy_docs/partner_docs/VisionXGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for VisionX'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | SIB Visions       |
+| **Partner Product**   |  VisionX       |
+| **Web Site**      |    https://visionx.sibvisions.com/   |
+| **Version & Platform**   |  VisionX 5.6.925 : CentOS7      |
```

Comment:
The version is ok.  It should be what we tested as that is when support starts for the integration.
The Platform should be what is supported by the integration.  Linux, windows, etc.  So what platforms does Sib Vision Vision X run on that we EDB would support the integration to run on?